ios/brave-ios/Sources/BraveStore/Subscription/SDK/BraveSkusSDK.swift

```diff
@@ -82,18 +82,15 @@ public enum BraveSkusEnvironment {
   case release
 
   /// Returns the current Skus SDK Environment
-  /// When DEBUG is defined, uses the Bundle-ID to determine the environment
-  /// Otherwise uses Brave-Core's version info.
   public static var current: BraveSkusEnvironment {
-    if AppConstants.buildChannel == .release {
+    if AppConstants.isOfficialBuild {
       return .release
     }
```

Comment:
Do you want beta/nightly builds to use release always? If not you need to remove this official build check

Reply:
Actually you're right. I don't need it. I was thinking it would be fine, but now that I think about it, we have to map this to the ProductID from the AppStore which is bound by the BundleID. So I can't change it.
So only way to point to a different env, should be via Brave-Core's switch.